### ISSUE TITLE
Upgrade to elm 0.19.1

### ISSUE THREAD

**Issue by mburri** [zwilias/elm-coverage#24]
Are there any plans to do an upgrade to elm 0.19.1?

I'm not able to use this package with elm 0.19.1 - it fails on installation:

elm version: 0.19.1-3
```
npm install --save-dev elm-coverage
```

Results in:
```
$ npm install --save-dev elm-coverage
npm WARN deprecated natives@1.1.6: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.

> elm@0.19.0 install /home/path/to/app/node_modules/elm-coverage/node_modules/elm
> binwrap-install

fs.js:27
const { Math, Object, Reflect } = primordials;
                                  ^

ReferenceError: primordials is not defined
    at fs.js:27:35
```

**Comment by zwilias** [comment]
Yeah, I'll look into that this weekend!

**Comment by lenards** [comment]
Is there anything I could do to help with this? 

I'm hitting same issue as @mburri. I know this tool is experimental, but I was considering it to measure coverage on the codebase I work on (about 27k SLoC) in 2020.

**Comment by zwilias** [comment]
Oh, dropped the ball here. Take 2: today!

**Comment by mburri** [comment]
I started too this morning and did a pull request #25 but the tests are failing. 
but i think its a starting point...

**Comment by zwilias** [comment]
Yeah, there are a bunch of extra changes that appear to be necessary, accounting for changes in elm-test and the output format of the compiler. Thanks for the PR, I just happened to already have made those changes locally when I saw your comment!

**Comment by zwilias** [comment]
0.19.1 support released in 0.3.0 🚀

**Comment by lenards** [comment]
Thank you Ilias! I'll check it out today.
